src/test/scala/akka/contrib/process/NonBlockingProcessSpec.scala

```diff
@@ -121,10 +138,9 @@ class NonBlockingProcessSpec extends WordSpec with Matchers with BeforeAndAfterA
   def expectDestruction(viaDestroy: Boolean): Unit = {
     val command = getClass.getResource("/sleep.sh").getFile
     new File(command).setExecutable(true)
-    val nameSeed = scala.concurrent.forkjoin.ThreadLocalRandom.current().nextLong()
     val streamProbe = TestProbe()
     val exitProbe = TestProbe()
-    val receiver = system.actorOf(Props(new NonBlockingReceiver(streamProbe.ref, exitProbe.ref, command, List.empty, nameSeed)), "receiver" + nameSeed)
+    val receiver = system.actorOf(Props(new NonBlockingReceiver(streamProbe.ref, exitProbe.ref, command, List.empty)))
```

Comment:
Can you detail why you removed the `nameSeed` parameter?

Reply:
The test I added failed initially because I'd copy-pasted the existing stdin test and I had a name clash with the actor from that test. 

So I added a second commit to remove the explicit actor names, so the akka system handles assigning non-clashing names - I didn't see a reason for assigning them explicitly. Although I don't claim to be an akka expert so happy to hear a reason otherwise...

Reply:
The only reason they were named is because things are then a bit easier to follow in the test log.

Reply:
It'd be great to keep the naming.